src/pretalx/cfp/views/user.py

```diff
@@ -397,6 +399,14 @@ def form_valid(self, form):
                     form.instance.update_duration()
                 if form.instance.pk and "track" in form.changed_data:
                     form.instance.update_review_scores()
+                if form.instance.pk and "additional_speaker" in form.changed_data:
+                    try:
+                        form.instance.send_invite(to=[form.cleaned_data.get('additional_speaker')],
+                                               _from=self.request.user)
+                    except SendMailException as exception:
+                        logging.getLogger("").warning(str(exception))
```

Comment:
Very bad usage of logger here.
A `logger` should be defined at the top of the file:

```py
logger = logging.getLogger(__name__)
```

The value passed to `getLogger()` is important, because later on, the log system can let you know where (which module) the message originated from.

Then use `logger` where you want to write message:

```py
logger.warning('Failed to send email with error: %s', exception)
```

Note: Don't build the string from the variables:

```py
logger.warning(f'Failed to send email with error: {exception}')
```

because it will always evaluate the variables regardless we want to emit messages or not.